src/ar/ARTester.cpp

```diff
@@ -1,105 +1,326 @@
-#include "Detector.h"
+#include <chrono>
+#include <ctime>
+#include <iostream>
+#include <string>
+#include <vector>
 
+#include <opencv2/calib3d.hpp>
 #include <opencv2/core.hpp>
+#include <opencv2/core/utility.hpp>
 #include <opencv2/highgui.hpp>
 #include <opencv2/imgproc.hpp>
 
-#include <iostream>
+#include "../ThreadedCapture.h"
+#include "Detector.h"
+
+// change to true to turn on timing code
+#define TIMING false
 
 const std::string ORIG_WINDOW_NAME = "Image (Original)";
-const std::string THRESH_TRACKBAR_NAME = "Threshold";
-const std::string THRESH2_TRACKBAR_NAME = "Threshold 2";
-const std::string BLUR_TRACKBAR_NAME = "Blur";
 
 constexpr bool EXTRA_WINDOWS = true;
 
+// Set to whichever camera params should be used
+AR::CameraParams PARAMS = AR::getCameraParams(AR::Params::ROBOT_TOP_WEBCAM_480);
+
+// Set to whichever MarkerSet should be used (CIRC/URC)
```

Comment:
Maybe we could mention that this can be overridden by command line params

Reply:
It's actually given a default value by the command line parser so on second thought it doesn't make too much sense to assign it a default value there. I think that comment is left over from before we had a command line parser and we would just hardcode those values in constants instead, so I'll remove that.